### PR TITLE
Release/0.5.0

### PR DESCRIPTION
**Pull request by BenFradet**

### CODE REVIEW

**Comment by BenFradet** [comment]
Yup, will do, thanks for the feedback

**Comment by alexanderdean** [comment]
This looks good. I guess my question is do we still need these resolvers in the README (I think you updated in the code):

https://github.com/snowplow/scala-forex#22-installation

**Comment by BenFradet** [comment]
Yup I'll clean this up.
Won't we still need a resolver for bintray? I'm not familiar with bintray.

**Comment by alexanderdean** [comment]
I pushed the updated OER_KEY... Hopefully this helps. Please reorder the commits as necessary and update the CHANGELOG! Thanks @BenFradet

**Comment by alexanderdean** [comment]
> Won't we still need a resolver for bintray though? I'm not familiar with bintray.

We shouldn't need it, because once we have an rc in Bintray, we can activate JCenter, and that will also mirror into Maven Central. I'm a little rusty on the details - @ninjabear can provide more details...

**Comment by BenFradet** [comment]
Ok, didn't know about the mirroring.

File: .travis/deploy.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+realm = Bintray API Realm
+host = api.bintray.com
+user = $BINTRAY_USER
+password = $BINTRAY_API_KEY
+EOF
+
+cd $TRAVIS_BUILD_DIR
+pwd
+
+project_version=$(sbt version -Dsbt.log.noformat=true | perl -ne 'print $1 if /(\d+\.\d+[^\r\n]*)/')
```
Comment (BenFradet) [inline .travis/deploy.sh]: It seems there is a problem with this line, since locally it gives 0.13/plugins0.5.0-M1

Comment (BenFradet) [inline .travis/deploy.sh]: I'll try removing support for x.x versioning

**Pull request merged**